index.html

```diff
@@ -0,0 +1,309 @@
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-4">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-5">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-6">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg btn-dark-yellow" href="#">See Project</a>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-section padding-left-right">
+        <div class="about-box">
+            <div class="about-text-box">
+                <h2 class="about-heading">About me</h2>
+                <p class="about-description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] In the `About section` you're missing the background image seen in the Figma design :+1: I will suggest that you fix that as well :wink: 
- Your Design
![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/981883b1-7b62-478d-bedb-b700e70c1a74)

- The Figma Design
![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/138d2154-ebdd-4a29-9540-e0e28aa0672a)